### PR TITLE
Test ascending

### PR DESCRIPTION
**Pull request by pierrejavelle**
new test for ascending values

### CODE REVIEW

**Review by feldmann-m — APPROVED** [review]
The test looks reasonable to me, however I cannot test it at the moment.

**Comment by aperezhortal** [comment]
Thanks @pierrejavelle for submitting the PR! I left a few review comments that would be great to tackle before merging. 
(Note that PR #300 is also open addressing the same issue.)

File: pysteps/tests/test_extrapolation_semilagrangian.py [hunk 1]
```diff
@@ -19,3 +19,13 @@ def test_semilagrangian():
...
     result = extrapolate(precip, velocity, num_timesteps)
     assert_array_almost_equal(result, expected)
+
+def test_ascending_time_step():
+    precip = np.ones((8, 8))
+    v = np.ones((8, 8))
+    velocity = np.stack([v, v])
+
+    not_ascending_timesteps = [1,2,3,5,4,6,7]
+    with pytest.raises(ValueError):
```
Comment (dnerini) [inline pysteps/tests/test_extrapolation_semilagrangian.py:33]: careful: the module `pytest` was not imported at the top

Comment (dnerini) [reply pysteps/tests/test_extrapolation_semilagrangian.py:33]: @pierrejavelle you only need to `import pytest` at top of this file and we'll be able to merge your pull request, please let me know if something is not quite clear.

**Pull request merged**